coala_quickstart/generation/FileGlobs.py

```diff
@@ -7,7 +7,7 @@
 from coalib.collecting.Collectors import collect_files
 
 
-def get_project_files(log_printer, printer, project_dir):
+def get_project_files(log_printer, printer, project_dir, noninteractive):
```

Comment:
Make `False` as the default value for `noninteractive`. That way the older tests will work as before. Then add a new test just to test `noninteractive=True`.

And maybe `non_interactive` is better than `noninteractive`? Since we use `args.non_interactive` ;)